firefoxbookmarks/src/extension.cpp

```diff
@@ -131,66 +132,72 @@ void FirefoxBookmarks::Private::finishIndexing() {
 /** ***************************************************************************/
 vector<shared_ptr<Core::StandardIndexItem>>
 FirefoxBookmarks::Private::indexFirefoxBookmarks() const {
+    // Build a new index
+    vector<shared_ptr<StandardIndexItem>> bookmarks;
 
-    QSqlDatabase database = QSqlDatabase::database(q->Core::Plugin::id());
+    {
```

Comment:
actually this extra stack is not really necessary. but is messes up the commit. please remove the scope.

Reply:
it won't cause any bugs (I guess), but http://doc.qt.io/qt-5/qsqldatabase.html#removeDatabase specifically points out, that you should keep any ressources associated to the database in a scope to destroy them before you remove the db

Reply:
oh okay, well then it is needed after all. thank you